deploy/monitor/prometheus_rule.yaml

```diff
@@ -0,0 +1,35 @@
+# Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    monitoring-key: middleware
+    prometheus: application-monitoring
+    role: alert-rules
+  name: application-monitoring
+spec:
+  selector:
+    matchLabels:
+      name: unifiedpush-operator
+  groups:
+    - name: general.rules
```

Comment:
This looks like a mixture of rules that cover both the unifiedpush-operator and the UnifiedPush Server. Is that intentional, or should those be in different PrometheusRule definitions?

Reply:
Yes, should be all in the same. Has no need to split them in POV.

Reply:
Yeah sure, it's probably fine, I don't know. I'm not familiar at all with the PrometheusRule type.

In a future release it might make sense to provision the PrometheusRule CR for the UnifiedPush Server from the operator :thinking: I would say that would wait for a future release though, since there are other concerns with that.

Reply:
Hi @grdryn,

IMHO: Make senses since both are the same type/object (PrometheusRules). The advantage to split them is given the flexibility to choose if we would like to both or just one of them applied.